main.rb

```diff
@@ -0,0 +1,74 @@
+require 'open3'
+
+def get_env_variable(key)
+  return (ENV[key] == nil || ENV[key] == "") ? nil : ENV[key]
+end
+
+def env_has_key(key)
+  value = get_env_variable(key)
+  return value unless value.nil? || value.empty?
+
+  abort("Input #{key} is missing.")
+end
+
+$output_path = env_has_key("AC_OUTPUT_DIR")
+$repo_path = env_has_key("AC_REPOSITORY_DIR")
+
+$npm_params = []
+if ENV["AC_RN_TEST_COMMAND_ARGS"] != "" && ENV["AC_RN_TEST_COMMAND_ARGS"] != nil
+  $npm_params = ENV["AC_RN_TEST_COMMAND_ARGS"].split("|")
```

Comment:
IMO `split` is not necessary, env should be str instead of array and should add after default jest params.

When use like above will it gives error?